changelog/fix-5525-cant-enable-cc-payments

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Corrected bug where checkbox remained disabled when credit card was disabled
```

Comment:
People who need this bug fixed will probably have sufficient context, but let's consider whether this message conveys all the needed information. Should it maybe mention the settings (page)? Perhaps replace "remained disabled" with "could not be enabled" or something similar?

Reply:
Good point. Fixed in https://github.com/Automattic/woocommerce-payments/pull/6504/commits/b4b286fd3a20da418ce5646f445fe46fa6a45ccd